src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Index.cshtml

```diff
@@ -67,7 +67,7 @@
                                     <h2 class="govuk-heading-m das-search-results__heading govuk-!-margin-bottom-0">
                                         <a href="@Url.Action("Detail", new { EncodedPledgeId = opportunity.ReferenceNumber })" class="das-search-results__link">@opportunity.EmployerName (@opportunity.ReferenceNumber)</a>
                                     </h2>
-                                    <p class="govuk-hint">up to @opportunity.DisplayAmount available in @Model.TaxYear</p>
```

Comment:
I'm guessing that the removal of this property reference in the markup leaves it unused, in which case we can remove it from the model as well.

Reply:
Done